src/utils/Utils.js

```diff
@@ -0,0 +1,15 @@
+export const testingFunction = () => {
+  console.log('it worked!');
+};
+
```

Comment:
I think this file should start with a lowercase letter, else it may get confused as a react component when people read it being imported.